website/docs/getting-started/features.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Features
+sidebar_position: 2
+---
+
+# Features provided by odo
+
+odo follows "create and push" workflow for almost everything. It means, as a user, when you "create" something the information (or manifest) is stored in a configuration file, and then upon doing a "push" it gets created  on the Kubernetes cluster.
+
+One can take an existing git repository and create an odo component from it, which can be pushed to a Kubernetes cluster.
```

Comment:
This phrase alone is strange. It is a consequence of the first paragraph, you could place it at the end of the first paragraph.

Reply:
Done.